lib/linter-utils.test.js

```diff
@@ -39,5 +39,65 @@ describe('Test verifies functionality of the BracketFilter Class', () => {
         expect(testStack.stackArray).toEqual([']', '{', '{']);
       });
 
+      it('test will create a class and return an array, input needs to be a BracketFilter Class', () => {
```

Comment:
nice work, Erik. Love that you went above and beyond and tested your application for these various scenarios, and your different approach to this problem.